Dockerfile

```diff
@@ -42,10 +43,16 @@ RUN for i in /tmp/apt/*.key; do apt-key add $i; done && \
 RUN /usr/bin/pip --no-cache-dir install awscli awsrequests testinfra && \
     /usr/bin/pip3 --no-cache-dir install awscli awsrequests testinfra
 
+# updated golang
+RUN wget https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz
+RUN tar -C /usr/local -xvf go1.12.7.linux-amd64.tar.gz
```

Comment:
Could you combine these into one RUN command, including the checksum, and add `rm  go1.12.7.linux-amd64.tar.gz`?  That'll let docker provide a smaller layer for this work.

If you move the go config section above the download, you could even add the `go get` for golint